ols/app/models/config.py

```diff
@@ -109,6 +109,56 @@ def validate_yaml(self) -> None:
             )
 
 
+class TLSConfig(BaseModel):
+    """TLS configuration."""
+
+    ssl_certfile: Optional[str] = None
+    ssl_keyfile: Optional[str] = None
+    ssl_keyfile_password: Optional[str] = None
+
+    def __init__(self, data: Optional[dict] = None) -> None:
+        """Initialize configuration and perform basic validation."""
+        super().__init__()
+        if data is not None:
+            self.ssl_certfile = data.get("ssl_certfile", self.ssl_certfile)
```

Comment:
This is IMO confusing, why not just say None for the default value?

Reply:
set to if data

Reply:
I meant
```
self.ssl_certfile = data.get("ssl_certfile", None)
```